legal-api/src/legal_api/models/naics_element.py

```diff
@@ -0,0 +1,63 @@
+# Copyright © 2022 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+"""NaicsElement are used to associate additional information to a given NAICS code at a given level.
+
+Element types can be examples, illustrative examples, inclusions and exclusions.
+"""
+from enum import Enum
```

Comment:
Can you use https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/utils/base.py

Simple demo https://github.com/thorwolpert/sql-enum

Reply:
done